JourneysSquad/JourneysSquad/Model/BillValidationError.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// An error that can be caused by incorrect input data for the bill.
+enum BillValidationError: Error {
+    /// Indicates too long participant's name.
```

Comment:
I thought we agreed to use the `person` word.

```suggestion
    /// Indicates too long person's name.
```

Reply:
Please review the rest of code.

Reply:
@dersim-davaod 
Is it regarding the name of a person? If I write about spendings, is it correct?

> Indicates that the sum of the bill doesn't equal the sum of all the participants's pays.

I think it doesn't make documentation hard readable, but expresses more clearly the essence of the sentence content.

Reply:
Fixed